cfme/services/catalogs/service_catalogs.py

```diff
@@ -1,8 +1,13 @@
 # -*- coding: utf-8 -*-
 from functools import partial
 
+from navmazing import NavigateToSibling, NavigateToAttribute
+
 import cfme.fixtures.pytest_selenium as sel
-from cfme.web_ui import accordion, flash, menu, form_buttons, Form, Input, Select
+from cfme.web_ui import accordion, flash, form_buttons, Form, Input, Select, match_location,\
```

Comment:
When the list gets this long, lets format it like below. Not something we've got everywhere yet, but as we're changing these lists its nice to format them consistently. 

```python
from cfme.web_ui import  (
    accordion, flash, blah, blah, blahto100chars,
    more_blah)
```